src/app.js

```diff
@@ -0,0 +1,8 @@
+console.info("I'm running...")
+
+import config from "./config/config.json" 
+console.log(config.name);
+
+import { User, Product } from "./models/models"
+const user = new User();
```

Comment:
Here you can only call `new User()` without assignment as it's only calls the constructor